src/content-handlers/iiif/modules/uv-openseadragoncenterpanel-module/OpenSeadragonCenterPanel.ts

```diff
@@ -366,24 +366,40 @@ export class OpenSeadragonCenterPanel extends CenterPanel<
     this.$zoomInButton.prop("aria-label", this.content.zoomIn);
     this.$zoomInButton.addClass("zoomIn viewportNavButton");
 
+    this.onAccessibleClick(this.$zoomInButton, () => {
+      this.zoomIn();
+    }, false, true);
```

Comment:
The buttons here all appear to be `div` elements; do we need the "anchor as button" setting?

Reply:
No they don't. Not sure why I did that here. Have removed.